distribution_rounds/round-1/distribution_results/raw_csv_exports/forum_post.md

```diff
@@ -0,0 +1,106 @@
+
+# TEC Rewards Distribution - round-1  - 11/07/21 to 30/07/21
+This period covers praise given between 11. July 2021 and 30. July 2021. We allocated 2546 TEC tokens for rewards, with a 100:0 split between Praise and Sourcecred. Some praise accounts still haven’t been activated so the total amount below will be less than what we set aside to distribute.
+
+Out of the total rewards:
+
+* 90% of the tokens were given as praise rewards :pray:
+* 7% distributed among quantifiers :balance_scale:
+* 3% assigned to the reward board :memo:
+
+This data has been reviewed by the Quantifiers and the Reward Board, and has been submitted for distribution to the [Reward Board DAO 1](https://xdai.aragon.blossom.software/#/rewardboardtec/)
+
+
+You can check out the full period analysis [here](ADD LINK HERE). :bar_chart:
+
+This post will be open to the community for review for 48 hours then submitted to the Reward Board for final execution. :heavy_check_mark:
+
+The Rewards Distribution for this round is as follows:
+|    | Username                    |   Rewards in TEC |
```

Comment:
seems to be a blank column on the left of this chart with the numbering, starting from 0 - we should remove this column